src/components/Modal/readme.md

```diff
@@ -708,3 +708,276 @@ class FormModal extends React.Component {
     </GlobalHeader>
 </div>
 ```
+
+##### make reservation modal
+
+```js
+import React from 'react';
+import { Modal, Button, Input, TimePicker, DatePicker, Textarea, Select } from 'react-rainbow-components';
+import { Field, reduxForm } from 'redux-form';
+import styled from 'styled-components';
+
+const Content = styled.div.attrs( props => ({
+    imgWidth: '328px'
+}))`
+    div:nth-child(2) img {
+        min-width: ${props => props.imgWidth};
+    }
+
+    @media (max-width: 768px) {
+        flex-direction: column-reverse;
+        
+        > div:nth-child(1) {
+            text-align: center;
+        }
+        
+        div:nth-child(2) img {
+            min-width: 0;
+            width: ${props => props.imgWidth};
+            max-width: 100%;
+        }
+    }
+`;
+
+const Title = styled.h2.attrs(props => {
+   return props.theme.rainbow.palette;
+})`
+    font-size: 36px;
+    font-weight: 300;
+    line-height: 'normal';
+    color: ${props => {
+        return (props.brand.main === '#80deea') ? '#fff' : '#061c3f';
+    }}
+`;
+
+const Description = styled.p.attrs(props => {
+   return props.theme.rainbow.palette;
+})`
+    font-size: 16px;
+    color: ${props => {
+            return (props.brand.main === '#80deea') ? '#fff' : 'rgba(0, 0, 0, 0.5)';
+    }}
+`;
+
+const countries = [
+    { value: '', label: 'Select country' },
+    { value: 'Mexico', label: 'Mexico' },
+    { value: 'United States of America', label: 'United States of America' },
+    { value: 'France', label: 'France' },
+    { value: 'Germany', label: 'Germany' },
+    { value: 'Argentina', label: 'Argentina' },
+    { value: 'Chile', label: 'Chile' },
+    { value: 'Colombia', label: 'Colombia' },
+    { value: 'China', label: 'China' },
+    { value: 'Brasil', label: 'Brasil' },
+    { value: 'Spain', label: 'Spain' },
+    { value: 'Greece', label: 'Greece' },
+    { value: 'Canada', label: 'Canada' }
+]; 
+
+function validate(values) {
+    const { date, time, email, country } = values;
+    const errors = {};
+    if (!date) {
+        errors.date = 'Date is a required field';
+    }
+    if (!time) {
+        errors.time = 'Time is a required field';
+    }
+    if (!email) {
+        errors.email = 'Email is a required field';
+    }
+    if (!country) {
+        errors.country = 'Country is a required field';
+    }
+    return errors;
+}
+
+function ReservationForm(props) {
+    const { handleSubmit, reset, onSubmit } = props;
+
+    const styles = {
+        input: {
+            marginTop: 24,
+        },
+        inputColumn: {
+            width: '48%',
+            marginTop: 24,
+        }
+    };
```

Comment:
since this is const styles that do not depend on props, it is better to put it outside the ReservationForm component since in each rerender a new object styles is recreated